core/db/entities.py

```diff
@@ -527,3 +527,12 @@ class ProjectFinanceChange(BaseModel):
             "programme_junction_id",
         ),
     )
+
+
+class GeospatialDim(BaseModel):
+    """Stores Geospatial information mapped to postcodes."""
+
+    __tablename__ = "geospatial_dim"
+
+    postcode_prefix = sqla.Column(sqla.String(), nullable=False, unique=True)
+    itl_region = sqla.Column(sqla.String(), nullable=False, unique=False)
```

Comment:
oh, sorry this wasn't on the ticket, but just realised we could add separate columns for itl1_region_code and itl1_region_name. Could you update this col and add the extra one?

Reply:
hmm or do we add the names to another normalised table? Worth thinking about...

Reply:
or another option is keep them both as cols in this table, but add a unique together constraint for codes and names

Reply:
If you go with both fields, it will be worth considering which field will actually be used by the filter, and adding an index to that one.